components/new-homepage/RaiseMoneySection.js

```diff
@@ -0,0 +1,156 @@
+            letterSpacing={[null, '-0.008em']}
+            lineHeight={['26px', '28px']}
+            textAlign={'center'}
+            color="black.800"
+            fontWeight="500"
+          >
+            <FormattedMessage
+              id="home.raiseMoney.description2"
+              defaultMessage={
+                "Whether you're a <u>mutual aid group</u>, a <u>community initiative</u>, an <u>open-source</u> <u>software project</u>, or a <u>climate change activist</u>, Open Collective helps groups unlock access to money."
+              }
+              values={{
+                u: I18nUnderline,
+              }}
+            />
+          </P>
+        </Box>
+        <Container
+          mt="72px"
+          width={['288px', '700px', '956px', null, '1200px']}
+          padding={['24px', '32px', '46px 56px']}
+          border="2px solid"
+          borderColor="primary.900"
+          borderRadius="16px"
+        >
+          <Box width={[null, '572px', '756px', null, '1000px']}>
+            <P
+              fontSize={['24px', '32px', '40px']}
+              lineHeight={['32px', '40px', '48px']}
+              letterSpacing={['-0.008em', null, '-0.04em']}
+              color="primary.900"
+              textAlign="left"
+              fontWeight="700"
+              mb={['16px', '24px']}
+            >
+              <FormattedMessage
+                id="home.stat"
+                defaultMessage={'Over 15,000 groups around the world raised and managed USD $35M last year'}
+              />
+            </P>
+          </Box>
+          <Box display="flex" alignItems={['center', 'flex-start']}>
+            <Box width={['184px', '572px', '756px']}>
+              <P
+                fontSize={['18px', '20px']}
+                letterSpacing={[null, '-0.008em']}
+                lineHeight={['26px', '28px']}
+                textAlign="left"
+                color="black.800"
+                fontWeight="500"
+              >
+                <FormattedMessage
+                  id="home.stat.checkout"
+                  defaultMessage={'Check out these amazing Collectives and connect with them.'}
+                />
+              </P>
+            </Box>
+            {/* TODO - Fix Arrow alignment on tab */}
+            <ArrowRight size={'40px'} color="#0C2D66" />
+          </Box>
+        </Container>
+      </Flex>
+      <Flex flexDirection="column" alignItems="center" mt="72px">
+        <Flex justifyContent="center" flexDirection={['column', 'row']} alignItem="center">
+          <Box width={['288px', '338px', '478px', null, '558px']} mr={[null, '24px']}>
+            <NextIllustration
+              width={558}
+              height={418}
+              src="/static/images/fiscal-hosting/tax-exempt.png"
+              alt="Fiscal hosting"
+            />
+          </Box>
+          <Box textAlign={['center', 'left']}>
+            <Link href="/fiscal-hosting">
+              <H3
+                as={'a'}
```

Comment:
A `<a>` within a `<a>` (from `Link`)? Sounds like this will generate an DOM nesting issue warning